tensorflow/python/data/experimental/kernel_tests/bucket_by_sequence_length_test.py

```diff
@@ -306,16 +422,18 @@ def _compute_batches(dataset):
             all_sparse_tensors.add(sprs_tensor)
       return all_sparse_tensors
 
-    dataset = _build_dataset()
-    boundaries = range(min_len + bucket_size + 1, max_len, bucket_size)
-    dataset = dataset.apply(grouping.bucket_by_sequence_length(
-        _element_length_fn,
-        boundaries,
-        [batch_size] * (len(boundaries) + 1),
-        no_padding=True))
-    batches = _compute_batches(dataset)
-    expected_batches = _compute_expected_batches()
-    self.assertEqual(batches, expected_batches)
+    for drop_remainder in (True, False):
+      dataset = _build_dataset()
+      boundaries = range(min_len + bucket_size + 1, max_len, bucket_size)
+      dataset = dataset.apply(grouping.bucket_by_sequence_length(
+          _element_length_fn,
+          boundaries,
+          [batch_size] * (len(boundaries) + 1),
+          no_padding=True,
+          drop_remainder=drop_remainder))
+      batches = _compute_batches(dataset)
+      expected_batches = _compute_expected_batches()
```

Comment:
Should this call take a `drop_remainder` argument?

Reply:
Yap, sorry about that, it slipped into the pylint commit 6842678